workers/unity/Packages/com.improbable.gdk.tools/DevAuthTokenUtils.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Improbable.Gdk.Tools.MiniJSON;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Tools
+{
+    public static class DevAuthTokenUtils
+    {
+        [MenuItem("SpatialOS/Generate Dev Authentication Token", false, MenuPriorities.GenerateDevAuthToken)]
+        private static void Generate()
+        {
+            string devAuthToken = string.Empty;
+            var devAuthTokenDir = GdkToolsConfiguration.GetOrCreateInstance().DevAuthTokenDir;
+            var devAuthTokenFilePath = Path.Combine(Application.dataPath, devAuthTokenDir, "DevAuthToken.txt");
+
+            var receivedMessage = string.Empty;
+            RedirectedProcess
+                .Command(Common.SpatialBinary)
+                .WithArgs("project", "auth", "dev-auth-token", "create", "--description", "\"Dev Auth Token\"",
+                    "--json_output")
+                .InDirectory(Common.SpatialProjectRootDir)
+                .AddOutputProcessing(message => receivedMessage += message)
```

Comment:
I believe we need to take the last message, not all of them. i.e. if you are on an old version of spatial or are not logged in, it will add additional log messages. this will screw up the deserialization later on